src/common/common.gypi

```diff
@@ -26,7 +26,12 @@
         },
       }],
       ['tizen == 1', {
-        'defines': ['TIZEN']
+        'defines': ['TIZEN'],
+        'variables': {
+          'packages': [
+            'dlog',
+          ],
+        },
```

Comment:
It's intended to be warning/error logger in release build. Debug messages will be ignored.